atom/browser/ui/cocoa/atom_menu_controller.mm

```diff
@@ -220,7 +220,8 @@ - (void)addItemToMenu:(NSMenu*)menu
     NSMenu* submenu = [[NSMenu alloc] initWithTitle:label];
     [item setSubmenu:submenu];
     [NSApp setServicesMenu:submenu];
-  } else if (type == atom::AtomMenuModel::TYPE_SUBMENU) {
+  } else if (type == atom::AtomMenuModel::TYPE_SUBMENU &&
```

Comment:
Do we know what the behavior is on Linux / Windows (with SimpleMenuModel)

Reply:
Yes, see linked code snippet in Chromium above